### ISSUE TITLE
Release of circe-fs2 - 1.4.1

### ISSUE THREAD

**Issue by otobrglez** [circe/circe-fs2#293]
Hey guys!

Are there any plans of releasing circe-fs2 that would work with Circe `0.14.1` which is the latest pre-0.15 release of Circe? 

That would make usage of circe-fs2 so nicer... 😅 

That's all. 

---

I tried changing the `circeVersion` in `build.sbt` to `0.14.1` and then running the testsuite and it fails because of circe-jawn dependency...

```
[error] sbt.librarymanagement.ResolveException: Error downloading io.circe:circe-jawn_sjs1_2.12:0.14.1
[error]   Not found
[error]   Not found
[error]   not found: /Users/otobrglez/.ivy2/local/io.circe/circe-jawn_sjs1_2.12/0.14.1/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/io/circe/circe-jawn_sjs1_2.12/0.14.1/circe-jawn_sjs1_2.12-0.14.1.pom
[error]         at lmcoursier.CoursierDependencyResolution.unresolvedWarningOrThrow(CoursierDependencyResolution.scala:258)
[error]         at lmcoursier.CoursierDependencyResolution.$anonfun$update$38(CoursierDependencyResolution.scala:227)
[error]         at scala.util.Either$LeftProjection.map(Either.scala:573)
[error]         at lmcoursier.CoursierDependencyResolution.update(CoursierDependencyResolution.scala:227)
[error]         at sbt.librarymanagement.DependencyResolution.update(DependencyResolution.scala:60)
[error]         at sbt.internal.LibraryManagement$.resolve$1(LibraryManagement.scala:59)
[error]         at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$12(LibraryManagement.scala:133)
[error]         at sbt.util.Tracked$.$anonfun$lastOutput$1(Tracked.scala:73)
[error]         at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$20(LibraryManagement.scala:146)
[error]         at scala.util.control.Exception$Catch.apply(Exception.scala:228)
[error]         at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$11(LibraryManagement.scala:146)
[error]         at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$11$adapted(LibraryManagement.scala:127)
[error]         at sbt.util.Tracked$.$anonfun$inputChangedW$1(Tracked.scala:219)
[error]         at sbt.internal.LibraryManagement$.cachedUpdate(LibraryManagement.scala:160)
[error]         at sbt.Classpaths$.$anonfun$updateTask0$1(Defaults.scala:3678)
[error]         at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error]         at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error]         at sbt.std.Transform$$anon$4.work(Transform.scala:68)
[error]         at sbt.Execute.$anonfun$submit$2(Execute.scala:282)
[error]         at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:23)
[error]         at sbt.Execute.work(Execute.scala:291)
[error]         at sbt.Execute.$anonfun$submit$1(Execute.scala:282)
[error]         at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error]         at sbt.CompletionService$$anon$2.call(CompletionService.scala:64)
[error]         at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error]         at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[error]         at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error]         at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
[error]         at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
[error]         at java.base/java.lang.Thread.run(Thread.java:832)
[error] (fs2JS / update) sbt.librarymanagement.ResolveException: Error downloading io.circe:circe-jawn_sjs1_2.12:0.14.1
[error]   Not found
[error]   Not found
[error]   not found: /Users/otobrglez/.ivy2/local/io.circe/circe-jawn_sjs1_2.12/0.14.1/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/io/circe/circe-jawn_sjs1_2.12/0.14.1/circe-jawn_sjs1_2.12-0.14.1.pom
[error] Total time: 23 s, completed 22 Sep 2021, 14:09:29
```
---

Thanks for making this lovely library! Cheers!

**Comment by sjmyuan** [comment]
We also encounter this problem, we are using renovate to update the dependencies automatically and most of the circe libraries published 0.14.1. The inconsistent version make the version management harder.

**Comment by armanbilge** [comment]
circe-fs2 0.14.0 is 100% compatible with any circe 0.14.x release. Please open an issue if you encounter any problems.
